spring-cloud-data-admin/src/main/java/org/springframework/cloud/data/admin/config/AdminConfiguration.java

```diff
@@ -86,7 +86,12 @@ public ModuleRegistry moduleRegistry() {
 	protected static class LocalConfig {
 
 		@Bean
-		public ModuleDeployer moduleDeployer(ModuleLauncher moduleLauncher) {
+		public ModuleDeployer longRunningModuleDeployer(ModuleLauncher moduleLauncher) {
```

Comment:
If we are considering these roles in a `@Qualifier`, then there's no need to reuse the bean name verbatim as the bean name (which doesn't look like a "role" anyways). So we could go with `@Qualifier("stream")` and `@Qualifier("task")` perhaps. Also notice I'm emphasizing 'stream' and 'task' as the context of their usage within spring-cloud-data, as opposed to 'LRP' and 'task' which are the terms specific to Diego.